awscli/topics/config-vars.rst

```diff
@@ -225,6 +225,47 @@ Example configuration::
   source_profile=development
 
 
+Support for Federated Users Using SAML 2.0
+------------------------------------------
+
+AWS CLI supports the use of federated user accounts through Security Assertion
+Markup Language (SAML) Identity Providers (IdP) which provides forms-based
+authentication.
+
+The AWS CLI will automatically authenticate you against the SAML IdP, and then
+retrieve AWS credentials for you. You do not need to configure any credentials.
+
+You can specify the following configuration values for configuring an SAML role
+in the AWS CLI config file:
+
+* ``saml_endpoint`` - The URL of the login page of your SAML IdP
+* ``saml_authentication_type`` - The authentication type of your SAML IdP.
+  So far, the only valid setting is `form`, means forms-based authentication.
```

Comment:
`form` --> ``form`` to make it a literal. Same thing for ``adfs`` and ``okta`` later in the guide.